reana_db/models.py

```diff
@@ -236,6 +236,12 @@ def get_quota_usage(self):
             cpu=self.get_user_quota_by_type(ResourceType.cpu),
         )
 
+    def has_exceeded_quota(self):
```

Comment:
I'm wondering if there should be a check per resource type as there might be some actions that the user can perform even when only the CPU quota is exceeded (create workflows, upload files, etc.) and the other way around (starting workflows).

Reply:
Decided offline to remain conservative and prevent the mentioned user actions when the quota of **any** user resource is exceeded.